src/v/crypto/test/crypto_test_utils.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+#include <boost/algorithm/hex.hpp>
+
+inline std::vector<uint8_t> convert_from_hex(const std::string& val) {
```

Comment:
this can just exist in the test source unless there are other future consumers in other files

Reply:
Other tests use this function